Packs/GCP/Integrations/GCP/GCP.yml

```diff
@@ -141,7 +141,391 @@ script:
+      description: Type of the resource (for example, compute#firewall).
+      type: String
+    - contextPath: GCP.Compute.Firewall.description
+      description: Description of the firewall rule.
+      type: String
+    - contextPath: GCP.Compute.Firewall.direction
+      description: Direction of traffic for the rule (INGRESS or EGRESS).
+      type: String
+    - contextPath: GCP.Compute.Firewall.disabled
+      description: Indicates whether the firewall rule is disabled.
+      type: Boolean
+    - contextPath: GCP.Compute.Firewall.priority
+      description: Priority value of the firewall rule.
+      type: Number
+    - contextPath: GCP.Compute.Firewall.network
+      description: The network URL this firewall rule applies to.
+      type: String
+    - contextPath: GCP.Compute.Firewall.selfLink
+      description: Server-defined URL for the resource.
+      type: String
+    - contextPath: GCP.Compute.Firewall.creationTimestamp
+      description: The creation timestamp of the firewall rule.
+      type: Date
+    - contextPath: GCP.Compute.Firewall.logConfig.enable
+      description: Indicates whether logging is enabled for the firewall rule.
+      type: Boolean
+    - contextPath: GCP.Compute.Firewall.sourceRanges
+      description: List of source IP ranges that the rule applies to.
+      type: Unknown
+    - contextPath: GCP.Compute.Firewall.targetTags
+      description: List of target instance tags to which the rule applies.
+      type: Unknown
+    - contextPath: GCP.Compute.FirewallNextToken
+      description: Next page token for pagination.
+      type: String
+  - name: gcp-compute-firewall-get
+    description: Retrieves a specific firewall rule by name.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: resource_name
+      required: true
+      description: Firewall rule name.
+    outputs:
+    - contextPath: GCP.Compute.Firewall.name
+      description: Firewall rule name.
+      type: string
+    - contextPath: GCP.Compute.Firewall.network
+      description: Network URL for the firewall rule.
+      type: string
+    - contextPath: GCP.Compute.Firewall.direction
+      description: Direction of traffic (INGRESS/EGRESS).
+      type: string
+    - contextPath: GCP.Compute.Firewall.priority
+      description: Priority of the rule.
+      type: number
+    - contextPath: GCP.Compute.Firewall.allowed
+      description: Allowed tuples.
+      type: Unknown
+    - contextPath: GCP.Compute.Firewall.denied
+      description: Denied tuples.
+      type: Unknown
+    - contextPath: GCP.Compute.Firewall.targetTags
+      description: Target instance tags.
+      type: Unknown
+  - name: gcp-compute-snapshots-list
+    description: Lists snapshots in the specified project.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: limit
+      description: Maximum number of results to return.
```

Comment:
Add the minimum and maximum values if written in the API documentation.
Update it in all the limit arguments.